server/sonar-server/src/main/java/org/sonar/server/computation/ComputationStepExecutor.java

```diff
@@ -19,70 +19,80 @@
  */
 package org.sonar.server.computation;
 
-import com.google.common.base.Throwables;
-import org.sonar.api.server.ServerSide;
-import org.sonar.api.utils.System2;
 import org.sonar.api.utils.log.Logger;
-import org.sonar.api.utils.log.Loggers;
 import org.sonar.core.util.logs.Profiler;
-import org.sonar.server.computation.activity.ActivityManager;
 import org.sonar.server.computation.monitoring.CEQueueStatus;
 import org.sonar.server.computation.step.ComputationStep;
-import org.sonar.server.computation.step.ComputationSteps;
 
 import static java.lang.String.format;
-import static org.sonar.db.compute.AnalysisReportDto.Status.FAILED;
-import static org.sonar.db.compute.AnalysisReportDto.Status.SUCCESS;
+import static java.util.Objects.requireNonNull;
 
-@ServerSide
-public class ComputationService {
-
-  private static final Logger LOG = Loggers.get(ComputationService.class);
-
-  private final ReportQueue.Item item;
-  private final ComputationSteps steps;
-  private final ActivityManager activityManager;
-  private final System2 system;
+public final class ComputationStepExecutor {
+  private final Logger logger;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)